File: app/src/main/java/com/gmail/fattazzo/formula1world/ergast/imagedb/objects/Status.java
package com.gmail.fattazzo.formula1world.ergast.imagedb.objects;

import com.activeandroid.Model;
import com.activeandroid.annotation.Column;
import com.activeandroid.annotation.Table;

/**
 * @author fattazzo
 *         <p/>
 *         date: 05/06/17
 */
@Table(name = "status")
public class Status extends Model {

    @Column
    String status;

    @Override
    public String toString() {
        return "Status{" +
                "statusid=" + getId() +
                ", status='" + status + '\'' +
                '}';
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/ergast/imagedb/objects/Status.kt
package com.gmail.fattazzo.formula1world.ergast.imagedb.objects

import com.activeandroid.Model
import com.activeandroid.annotation.Column
import com.activeandroid.annotation.Table

/**
 * @author fattazzo
 *
 *
 * date: 05/06/17
 */
@Table(name = "status")
class Status : Model() {

    @Column
    internal var status: String? = null

    override fun toString(): String {
        return "Status{" +
                "statusid=" + id +
                ", status='" + status + '\'' +
                '}'
    }
}
